gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/JobStatistics.java

```diff
@@ -61,7 +61,7 @@ private ExtractStatistics(Builder builder) {
     /**
      * Returns the number of files per destination URI or URI pattern specified in the extract job.
      * These values will be in the same order as the URIs specified by
-     * {@link ExtractJobInfo#destinationUris()}.
+     * {@link ExtractJobConfiguration#destinationUris()}.
```

Comment:
Unrelated to this PR: are the 3 `@SuppressWarnings("unchecked")`on fromPb necessary?